hydroflow_datalog_core/examples/voting/leader.dl

```diff
@@ -0,0 +1,11 @@
+.input clientIn
```

Comment:
If these aren't being used in any actual tests we should probably drop them for now.